### PR TITLE
Pattern Library V2 - Update to reflect the current brand.

### PR DESCRIPTION
**Pull request by mattheu**

### CODE REVIEW

File: assets/src/scripts/components/SearchBar/SearchBar.js [hunk 1]
```diff
@@ -19,40 +19,40 @@ export default class SearchBar extends Component {
 	render() {
 
 		let searchBarClassNames = classNames({
-			'SearchBar':            true,
-			'SearchBar-Focused':    this.state.focused,
-			'SearchBar-HasResults': this.state.focused,
-			'SearchBar-Loading':    this.state.loading,
+			'search-bar':            true,
```
Comment (roborourke) [inline assets/src/scripts/components/SearchBar/SearchBar.js]: Indentation is a little funky, mix of tabs & spaces?

File: assets/src/scripts/components/SearchBar/SearchBar.js [hunk 2]
```diff
@@ -97,13 +96,12 @@ export default class SearchBar extends Component {
...
 			}
 		}).then( ( json ) => {
-
 			this.setState( { results: json.results } );
 
 			// Delay disable loading just a little.
 			window.setTimeout( () => {
 				this.setState( { loading: false } );
-			}, 500 );
+			}, 5000000 );
```
Comment (roborourke) [inline assets/src/scripts/components/SearchBar/SearchBar.js]: debug code?

**Comment by roborourke** [comment]
Also just recommit & clobber the `package-lock.json` once you regenerate it

**Comment by shadyvb** [comment]
@roborourke @mattheu Are we missing an updated submodule here by any chance ?

**Comment by shadyvb** [comment]
^^ Issued #99 for the submodule update.

**Pull request merged**